src/drivers/cloudflare-kv-http.ts

```diff
@@ -86,7 +87,7 @@ type CloudflareAuthorizationHeaders =
 
 const DRIVER_NAME = "cloudflare-kv-http";
 
-export default defineDriver<KVHTTPOptions>((opts) => {
+const driver: DriverFactory<KVHTTPOptions, void> = defineDriver((opts) => {
```

Comment:
this should probably be `never`, in that it doesn't have a `getInstance`.

```suggestion
const driver: DriverFactory<KVHTTPOptions, never> = defineDriver((opts) => {
```